imsv-docs-astro/src/redirects.yml

```diff
@@ -35,6 +35,7 @@
 /guides/custodial wallets/card-lifecycle: "/guides/card-lifecycle"
 /guides/custodial wallets/physical-cards: "/guides/issue-a-virtual-card"
 /guides/custodial-wallets: "/use-cases/custodial-wallets"
+/guides/complete-headless-kyc-checks: "guides/partner-conducted-kyc"
```

Comment:
please sort the lines

Reply:
What do you mean by that ?

Reply:
I have changed the order to have it sorted alphabetically, I guess this is what you meant by sort the lines. @ncjones